### PR TITLE
Add validation tooltip to mailserver/bootnode address input

### LINKED ISSUE #7752: No error message if provide wrong mailserver or bootnode url
### Description

[comment]: # (Feature or Bug? i.e Type: Bug)
*Type*: Bug

[comment]: # (Describe the feature you would like, or briefly summarise the bug and what you did, what you expected to happen, and what actually happens. Sections below)
*Summary*: There is no error message if provided url is considered as invalid. Save button is not enabled and user has no clues on what's wrong and what is expected format (e.g. as in https://github.com/status-im/status-react/issues/7746)

We can show error above the filed with a message on what's wrong and share some expected format for url... Or can add a link to tech doc where adding custom mailserver and sample of the  url format/example are provided...

#### Expected behavior
[comment]: # (Describe what you expected to happen.)
Explanation on why the provided url is invalid and what's the valid format
#### Actual behavior
[comment]: # (Describe what actually happened.)
No error message
### Reproduction
[comment]: # (Describe how we can replicate the bug step by step.)
For mailserver:
- Open Status -> Profile -> Advanced -> Mailserver 
- Turn off automatic selection 
- Tap + to add custom mailserver
- provide Name and type "invalid" into mailserver address field
- Check Save button and if there is any error message

IMG_5969

For bootnode:
- enable bootnode in Profile -> Advanced -> Bootnodes
- sign in and go again to Profile -> Advanced -> Bootnodes
- Tap + to add bootnode
- provide Name and type "invalid" into bootnode address field

### Additional Information
[comment]: # (Please do your best to fill this out.)
* Status version: all previous releases and current nightly
* Operating System: Android and iOS

#### Logs

[comment]: # (if on Android please replicate bug whilst running adb logcat)
```
...
```

### PR DESCRIPTION
**Pull request by rasom**
fix #7752

status: ready

### CODE REVIEW

File: src/status_im/ui/screens/offline_messaging_settings/edit_mailserver/views.cljs [hunk 1]
```diff
@@ -60,14 +62,21 @@
...
+           [text-input/text-input-with-label
+            {:label          (i18n/label :t/mailserver-address)
+             :placeholder    (i18n/label :t/specify-mailserver-address)
+             :content        qr-code
+             :style          styles/input
+             :container      styles/input-container
+             :default-value  url
+             :on-change-text #(re-frame/dispatch [:mailserver.ui/input-changed :url %])}]
+           (when (and (not (string/blank? url)) (not is-valid?))
+             [tooltip/tooltip (i18n/label :t/invalid-format)
```
Comment (cammellos) [inline src/status_im/ui/screens/offline_messaging_settings/edit_mailserver/views.cljs]: enode and mailserver don't have the same format, mailserver wants a password:
`enode://{enode-id}:{password}@{ip-address}:{port}` 
so we might want different messages

Comment (rasom) [reply src/status_im/ui/screens/offline_messaging_settings/edit_mailserver/views.cljs]: sure, thanks

Comment (rasom) [reply src/status_im/ui/screens/offline_messaging_settings/edit_mailserver/views.cljs]: so for bootnode it is `enode://{enode-id}@{ip-address}:{port}`, correct?

Comment (cammellos) [reply src/status_im/ui/screens/offline_messaging_settings/edit_mailserver/views.cljs]: yes, I believe so

Comment (rasom) [reply src/status_im/ui/screens/offline_messaging_settings/edit_mailserver/views.cljs]: ok, fixed

**Review by cammellos — APPROVED** [review]

**Review by mandrigin — APPROVED** [review]

**Comment by churik** [comment]
@rasom 
1) Can add mailserver without a Name if you don't set focus on Mailserver address directly 
However, it is the same in nightly, so will be fixed separately?

2) Can get invaild format on Address with empty name (mailservers and bootnodes)

Steps:
- Go to mailserver settings
- Tap on +
- Type smth in Name and erase it (you can do it after or before pasting valid mailserver address)
- Scan or paste valid address to Mailserver address.
Expected result: no error, can't save mailserver without name ("Name is required" error is shown)
Actual result: Invalid format error

**Comment by rasom** [comment]
@churik I haven't touched the logic at all, I just added tooltips and different placeholders for inputs. This PR is only about giving a user some feedback on invalid input.

**Comment by churik** [comment]
@rasom I agree that 1 should be reported and fixed separately.

According to 2 - on nightly when you paste right address URL in bootnode/mailserver address - there is no tooltip, so I can change fields until "Save" button will be available. On PR build even address is valid, but name field is empty - the error is shown on address field and it overlaps empty name field, so I can't easily fix it (I need firstly to delete value for address).
So I think that 2 should be fixed here

**Comment by rasom** [comment]
@churik fix pushed

**Comment by churik** [comment]
@rasom probably rebase to latest develop is needed...nix PR was merged, so that can't be built (I think it can be the reason)

**Comment by rasom** [comment]
@churik done

**Comment by churik** [comment]
Tested on IOS 11.4.1 and Android 8.
Addding / editing / removing custom mailservers and bootnodes.
On desktop there is no custom mailserver / bootnodes, so it doesn't affect to desktop builds.

**Pull request merged**